Ports/imgcat/package.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=imgcat
 version=2.5.0
-useconfigure=false
-files="https://github.com/eddieantonio/imgcat/releases/download/v${version}/imgcat-${version}.tar.gz imgcat-v${version}.tar.gz 1b5d45ccafc6fbb7a7ee685d4a5110101418d48b"
+depends="ncurses"
+files="https://github.com/eddieantonio/imgcat/releases/download/v${version}/imgcat-${version}.tar.gz imgcat-v${version}.tar.gz 16e5386f5ce2237643785e7c8bb2cfce"
 auth_type=md5
 
-depends="ncurses"
+build() {
+    run make \
+        production=true \
+        CURSES_CFLAGS="-I${SERENITY_ROOT}/Build/i686/Root/usr/local/include/ncurses"
```

Comment:
Please don't hardcode `Build/i686`, use `SERENITY_BUILD_DIR` instead. Also, why is both `CURSES_CFLAGS` and `CPPFLAGS` needed?

Reply:
I tried to splice parts from different ports as I was trying to debug the problem properly. I'll remove the former and remove the hardcoded thing, and edit it out of other places in the ports as well.